docs/sources/layers/core.md

```diff
@@ -7,6 +9,19 @@ keras.layers.core.Layer()
 __Methods__:
 
 ```python
+__call__
+```
+
+Apply Layer to an external input tensor.
+
+- __Return__: Tensor.
+
+- __Arguments__:
+    - __X__: Tensor. External input Tensor that will temporally replace
```

Comment:
How about "Apply layer transformation defined by its `get_output` method to an input `X`." Do you have any suggestion?